wagtail/admin/tests/pages/test_preview.py

```diff
@@ -360,22 +521,6 @@ def test_disable_preview_on_create(self):
         self.assertNotContains(response, 'data-action="%s"' % preview_url)
 
     def test_disable_preview_on_edit(self):
-        simple_page = SimplePage(title="simple page", content="hello")
-        self.root_page.add_child(instance=simple_page)
-
-        # preview button is available by default
-        response = self.client.get(
-            reverse("wagtailadmin_pages:edit", args=(simple_page.id,))
-        )
-        self.assertEqual(response.status_code, 200)
-
-        preview_url = reverse(
-            "wagtailadmin_pages:preview_on_edit", args=(simple_page.id,)
-        )
-        self.assertContains(response, 'data-side-panel-toggle="preview"')
-        self.assertContains(response, 'data-side-panel="preview"')
-        self.assertContains(response, 'data-action="%s"' % preview_url)
-
```

Comment:
I moved the assertions that ensure "preview button is available by default" to separate tests in `TestEnablePreview`, because I think it's more suitable than having these inside tests that ensure preview can be disabled.